test/e2e/storage/utils/utils.go

```diff
@@ -59,177 +59,27 @@ const (
 	// KStop defines stop value
 	KStop KubeletOpt = "stop"
 	// KRestart defines restart value
-	KRestart KubeletOpt = "restart"
+	KRestart     KubeletOpt = "restart"
+	minValidSize string     = "1Ki"
+	maxValidSize string     = "10Ei"
 )
 
 const (
 	// ClusterRole name for e2e test Priveledged Pod Security Policy User
 	podSecurityPolicyPrivilegedClusterRoleName = "e2e-test-privileged-psp"
 )
 
-// PodExec runs f.ExecCommandInContainerWithFullOutput to execute a shell cmd in target pod
-func PodExec(f *framework.Framework, pod *v1.Pod, shExec string) (string, string, error) {
-	if framework.NodeOSDistroIs("windows") {
-		return f.ExecCommandInContainerWithFullOutput(pod.Name, pod.Spec.Containers[0].Name, "powershell", "/c", shExec)
-	}
-	return f.ExecCommandInContainerWithFullOutput(pod.Name, pod.Spec.Containers[0].Name, "/bin/sh", "-c", shExec)
-
-}
-
-// VerifyExecInPodSucceed verifies shell cmd in target pod succeed
-func VerifyExecInPodSucceed(f *framework.Framework, pod *v1.Pod, shExec string) {
-	stdout, stderr, err := PodExec(f, pod, shExec)
-	if err != nil {
-
-		if exiterr, ok := err.(uexec.CodeExitError); ok {
-			exitCode := exiterr.ExitStatus()
-			framework.ExpectNoError(err,
-				"%q should succeed, but failed with exit code %d and error message %q\nstdout: %s\nstderr: %s",
-				shExec, exitCode, exiterr, stdout, stderr)
-		} else {
-			framework.ExpectNoError(err,
-				"%q should succeed, but failed with error message %q\nstdout: %s\nstderr: %s",
-				shExec, err, stdout, stderr)
-		}
-	}
-}
-
 // VerifyFSGroupInPod verifies that the passed in filePath contains the expectedFSGroup
```

Comment:
how to decide which function goes to utils, and which one goes to e2evolume?

Reply:
The principles I am using here are:
1. If this function has only been called by e2e/storage/..., it should go to e2e/storage/utils, because no other component is using it so lets just keep it at the lowest level
2. If there is any function that has been called outside of storage, it should go to a more general location namely e2evolume or e2epod.

In addition to this, dependency management here is that subpackage can import corepackage, but core cannot import sub.
That being said, previously e2evolume import e2e/storage/utils which is not the right direction. So I move these function up to the e2evolume to demove clear the dependency. 

I think this direction of dependency also align with another effort from sig testing. Does this make sense?